### PR TITLE
new(support): create `SUPPORT.md` file

### PR DESCRIPTION
**Pull request by DerekNonGeneric**
Lets people know about ways to get help with a project.

Refs: https://help.github.com/en/github/building-a-strong-community/adding-support-resources-to-your-project

Signed-off-by: Derek Lewis <DerekNonGeneric@inf.is>

/cc @septs @smorimoto @jorgebucaran @yuvilio @emmitrovic as there is a link on the readme now :)

### CODE REVIEW

**Comment by DerekNonGeneric** [comment]
I think I would much prefer this other one than the official one that I have put there&hellip;

File: SUPPORT.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+on the _matrix.org_ network. It’s a synchronous chat channel for lightweight
+communication. Good uses of it are for asking questions, casual conversation,
+collaboration, and quick exchanges.
+
+**[@openinf][twitter-account] on Twitter:** The OpenINF support account on
+Twitter.
+
+[docs-site]: https://open.inf.is/
+[matrix-channel]: https://matrix.to/#/#openinf:matrix.org
+[twitter-account]: https://twitter.com/openinf
```
Comment (septs) [inline SUPPORT.md:22]: suggest, unified brand and username as one

https://github.com/OpenINF
https://twitter.com/OpenINF

Comment (DerekNonGeneric) [reply SUPPORT.md:22]: I am trying to remember why, but when I had it like that before, it caused a big headache for the GitHub one; let me think about that and try to remember what the issue was&hellip;

Comment (DerekNonGeneric) [inline SUPPORT.md:22]:
```suggestion
[twitter-account]: https://twitter.com/OpenINF
```

Comment (DerekNonGeneric) [reply SUPPORT.md:22]:
```suggestion
[twitter-account]: https://twitter.com/OpenINF
```

File: SUPPORT.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+**[`#openinf`][matrix-channel] on matrix.org:** The OpenINF Matrix support channel
+on the _matrix.org_ network. It’s a synchronous chat channel for lightweight
+communication. Good uses of it are for asking questions, casual conversation,
+collaboration, and quick exchanges.
+
+**[@openinf][twitter-account] on Twitter:** The OpenINF support account on
+Twitter.
+
+[docs-site]: https://open.inf.is/
```
Comment (DerekNonGeneric) [inline SUPPORT.md:20]:
```suggestion
[docs-site]: https://open.inf.is/en/docs
```

Comment (septs) [reply SUPPORT.md:20]: suggest, read the https://developers.google.com/search/docs/advanced/crawling/managing-multi-regional-sites

File: SUPPORT.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+**[open.inf.is][docs-site] on the Web:** The OpenINF website. It has docs and
+tutorial resources.
+
+**[`#openinf`][matrix-channel] on matrix.org:** The OpenINF Matrix support channel
+on the _matrix.org_ network. It’s a synchronous chat channel for lightweight
+communication. Good uses of it are for asking questions, casual conversation,
+collaboration, and quick exchanges.
+
+**[@openinf][twitter-account] on Twitter:** The OpenINF support account on
```
Comment (DerekNonGeneric) [inline SUPPORT.md:17]:
```suggestion
**[@OpenINF][twitter-account] on Twitter:** The OpenINF support account on
```

File: SUPPORT.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+Twitter.
```
Comment (DerekNonGeneric) [inline SUPPORT.md:18]:
```suggestion
Twitter. Be sure to follow the [Twitter Rules] and our [Code of Conduct].
```

File: SUPPORT.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+[docs-site]: https://open.inf.is/docs
+[matrix-channel]: https://matrix.to/#/#openinf:matrix.org
+[twitter-account]: https://twitter.com/OpenINF
```
Comment (DerekNonGeneric) [inline SUPPORT.md:22]:
```suggestion
<!-- prettier-ignore-start -->
<!-- PRESERVE LINK DEFINITION LABEL CASE - START -->

[Twitter Rules]:
  https://help.twitter.com/en/rules-and-policies/twitter-rules
  'The Twitter rules: safety, privacy, authenticity, and more'

[Code of Conduct]:
  ./CODE_OF_CONDUCT.md
  'Standards for how to engage with the project community'

<!-- PRESERVE LINK DEFINITION LABEL CASE - END -->
<!-- prettier-ignore-end -->
```

File: SUPPORT.md (hunk 4 above)
Comment (septs) [reply SUPPORT.md:18]:
```html
<abbr title="Direct Messages">DMs</abbr>
```

> <abbr title="Direct Messages">DMs</abbr>

but github markdown render engine non-supported.

File: SUPPORT.md (hunk 5 above)
Comment (septs) [reply SUPPORT.md:22]: suggest, use markdownlint toolchain

## library

see https://github.com/DavidAnson/markdownlint

## cli tool

for gh actions:

```yaml
- name: Run markdown checking
  run: npx markdownlint-cli2 "**/*.md"
```

see https://github.com/DavidAnson/markdownlint-cli2

## vscode extension

see https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint

File: SUPPORT.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+## Resources
+
+**[open.inf.is][docs-site] on the Web:** The OpenINF website. It has docs and
+tutorial resources.
+
+**[`#openinf`][matrix-channel] on matrix.org:** The OpenINF Matrix support channel
+on the _matrix.org_ network. It’s a synchronous chat channel for lightweight
+communication. Good uses of it are for asking questions, casual conversation,
+collaboration, and quick exchanges.
```
Comment (DerekNonGeneric) [inline SUPPORT.md:15]:
```suggestion
collaboration, and quick exchanges. Be sure to follow our [Code of Conduct].
```

File: SUPPORT.md (hunk 4 above)
Comment (DerekNonGeneric) [reply SUPPORT.md:18]: That's alright, these will end up being processed by another Markdown process (not necessarily the one GitHub for GitHub-flavored Markdown).

File: SUPPORT.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+## Resources
+
+**[open.inf.is][docs-site] on the Web:** The OpenINF website. It has docs and
+tutorial resources.
+
+**[`#openinf`][matrix-channel] on matrix.org:** The OpenINF Matrix support
+channel on the _matrix.org_ network. It’s a synchronous chat channel for
+lightweight communication. Good uses of it are for asking questions, casual
+conversation, collaboration, and quick exchanges. Be sure to follow our [Code of
+Conduct].
```
Comment (DerekNonGeneric) [inline SUPPORT.md:16]:
```suggestion
Conduct](./CODE_OF_CONDUCT.md).
```

File: SUPPORT.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+**[@OpenINF][twitter-account] on Twitter:** The OpenINF support account on
+Twitter. <abbr title="Direct Messages">DMs</abbr> (Direct Messages) are open. Be
+sure to follow the [Twitter Rules] and our [Code of Conduct].
```
Comment (DerekNonGeneric) [inline SUPPORT.md:20]:
```suggestion
sure to follow the [Twitter Rules][twitter-rules] and our [Code of Conduct](./CODE_OF_CONDUCT.md).
```

File: SUPPORT.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+
+[docs-site]: https://open.inf.is/docs
+[matrix-channel]: https://matrix.to/#/#openinf:matrix.org
+[twitter-account]: https://twitter.com/OpenINF
```
Comment (DerekNonGeneric) [inline SUPPORT.md:24]:
```suggestion
[twitter-account]: https://twitter.com/OpenINF
[twitter-rules]: https://help.twitter.com/en/rules-and-policies/twitter-rules
```

File: SUPPORT.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+**[`#openinf`][matrix-channel] on matrix.org:** The OpenINF Matrix support
+channel on the _matrix.org_ network. It’s a synchronous chat channel for
+lightweight communication. Good uses of it are for asking questions, casual
+conversation, collaboration, and quick exchanges. Be sure to follow our [Code of
+Conduct].
+
+**[@OpenINF][twitter-account] on Twitter:** The OpenINF support account on
+Twitter. <abbr title="Direct Messages">DMs</abbr> (Direct Messages) are open. Be
+sure to follow the [Twitter Rules] and our [Code of Conduct].
+
```
Comment (DerekNonGeneric) [inline SUPPORT.md:21]:
```suggestion
[coc]: ./CODE_OF_CONDUCT.md
```

File: SUPPORT.md [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+**[`#openinf`][matrix-channel] on matrix.org:** The OpenINF Matrix support
+channel on the _matrix.org_ network. It’s a synchronous chat channel for
+lightweight communication. Good uses of it are for asking questions, casual
+conversation, collaboration, and quick exchanges. Be sure to follow our [Code of
+Conduct][coc].
+
+**[@OpenINF][twitter-account] on Twitter:** The OpenINF support account on
+Twitter. <abbr title="Direct Messages">DMs</abbr> (Direct Messages) are open. Be
+sure to follow the [Twitter Rules][twitter-rules] and our [Code of Conduct][coc].
+[coc]: ./CODE_OF_CONDUCT.md
```
Comment (DerekNonGeneric) [inline SUPPORT.md:21]:
```suggestion
sure to follow the [Twitter Rules][twitter-rules] and our [Code of Conduct][coc].

[coc]: ./CODE_OF_CONDUCT.md
```

**Pull request merged**